src/strings/PowerFxResources.en-US.resx

```diff
@@ -4464,4 +4464,23 @@
       A partial operator is the 2nd part of a statement `[Partial Op]` and can be one of "And", "Or", "Table" or "Record".
       It's used to determine how to combine multiple expressions with the same name and operator.</comment>
   </data>
+  <data name="ErrNeedTable" xml:space="preserve">
+    <value>Argument needs to be a table.</value>
```

Comment:
>Argument needs to be a table. [](http://example.com/codeflow?start=11&length=29)

We already have an error that we can use for this scenario. If you run something like `CountRows({a:1})` you should get it.